scripts/annotrace.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
```

Comment:
This file should not be needed, we already have an annotate script. Unless this does something the existing one doesn't handle? @LawrenceEsswood

Reply:
Where is the alternative so I can compare?

Reply:
scripts/symbolize-cheri-trace.py

Reply:
Looks to do roughly the same thing, minus a few small differences. symbolize-cheri-trace.py seems to use the wrong arguments for addr2line (which it uses if LLVM_SYMBOLIZER is not provided). Also, anno-trace goes and opens source files to get source annotations. Dunno if that is actually needed, or if I just didn't appreciate how much debug info was available through standard utilities.